lib/order.rb

```diff
@@ -0,0 +1,86 @@
+
+require 'csv'
+require_relative "customer"
+
+class Order
+
+  attr_reader :id, :customer, :fulfillment_status
+  attr_accessor :products
+
+    def initialize(id, products  , customer, fulfillment_status = :pending)
```

Comment:
Your indentation gets off here. It's not syntactically important, but consistent indentation makes it much easier to find bugs.